src/dbt/kipptaf/models/exposures/tableau.yml

```diff
@@ -559,6 +559,21 @@ exposures:
           metadata:
             id: b11542a2-6780-4feb-a1dd-534ffe724faf
             cron_schedule: 0 * * * *
+  - name: personalized_survey_links
+    label: Personalized Survey Links
+    type: dashboard
+    owner:
+      name: Data Team
+    depends_on:
+      - ref("rpt_tableau__survey_completion")
+    meta:
+      dagster:
+        kinds:
+          - tableau
+        asset:
+          metadata:
+            id: 
```

Comment:
@cbini need your help filling in the id for this workbook, it's a live data source right now but should be hourly like the Survey Dashboard https://tableau.kipp.org/#/site/KIPPNJ/workbooks/5683/datasources